### ISSUE TITLE
Gtk: Feature Discussion Quote of the Day/ News Fetcher

### ISSUE THREAD

**Issue by TheShadowOfHassen** [olivierkes/manuskript#1112]
So I've noticed that in my brief time in helping with bug reports there have been problems with non up to date versions of manuskript. And I know that the hole Linux distributions will have whatever version of manuskript, and we can't stop that, but I thought it would still be useful to have something that could notify someone of a new update or give other news.

I mulled over the idea and then came up with this plan.

So what if the general tab was renamed overview and was used for more than just the user's name/ email etc. Beneath it could be added a news display which would have to be implemented by some sort of rss feed and gtkwebview but what if you were offline.

Then it would display something like this:

![Jane_Austen markup](https://user-images.githubusercontent.com/94244235/211222335-0533a883-9357-4dec-91fa-13e8a57ede32.jpg)

I just did that in gimp. In reality, it would be some sort of image/ label overlay. I have a fragment of the UI in my pull request (#1111) It's based off of Lollypop’s full screen UI. I can implement it and will once the other changes are merged, but I wanted to know if anyone else had any ideas/ feedback.

I also thought that the latest source of manuskript there is a target window; the basics of that could also be implemented in some sort of banner which could pop up saying things like: "1,000 words this session. Keep it up!"

It isn't that radical of a design change and I think It'd be a nice implementation. The only problem is the manuskript website doesn't have a rss feed. But I was planning on just implementing the quotes for the time being. The only problem is we'd have to stick for long dead authors because I looked for an image for C.S. Lewis because I have some very good writing quotes from him but all the images were still copyrighted.

**Comment by TheShadowOfHassen** [comment]
So another idea/ revision of the idea. Maybe not make it look like a bad meme and skip the images. That would make it slightly less cool but also save storage and probably make it easier for all screen (and we'd be able to use quotes that we couldn't find images for.) Unless anyone says otherwise I'll probably just do that.

Also I was wondering, maybe a pop up banner like LibreOffice has between the file menu and the rest of the text. That could be used for the great job keep it up! and also update/ news notifying. This feature will probably have to come in three parts. First part with the quotes. Second part when the Gtk version gets editing/ saving I can do the streak monitor and finally the news fetcher.

I am posting all of this because I am open for feedback and feature requests for this idea. I think its a cool idea but I don't want to implement anything that would be very annoying. Plus the more people who give their favorite writing quotes the better.

Anyone can comment their favorite writing quotes and I'll add them, to my list.

**Comment by TheJackiMonster** [comment]
I personally don't really mind these banners in GIMP, Krita, Blender and so on. From my perspective it's just important they don't impact quick usage. Adding time between starting an application and using it can be frustrating to users. So we should definitely make it optional as many other applications do (using a checkbox with "Don't show again" for example).

But I'm not against the idea of adding something like that because it can make the application more welcoming to new users, I assume.

**Comment by TheShadowOfHassen** [comment]
Settings would be for anything. Settings whether you want any of this stuff, settings for whether you just want news on manuksript updates or other news and everything else.. (There isn't other news now but I think a manuskript writing competition would be cool and that would be news) I don't mean a start up banner just a little thing that you can close up on the top bar.

**Comment by TheShadowOfHassen** [comment]
So I know how I'll do this, but the only question is what to do about the quote managing. I think just a text file in resources would work for keeping them but the quotes will need moderated  to make certain they are reliant and also not offensive. I can do that but do you want me to keep a different repository for them and every release update them or just should we have people make pull request and they just are approved? (That couldn't be me because I can't approve pull requests but I can check them over and comment that I like them and then the pull requests can be aproved.

**Comment by TheShadowOfHassen** [comment]
Yes, I completely forgot about that although that wouldn't be to hard just have different files correlating to different languages and it reads whichever UI language it reads. (For the quotes) For the news and "Good Job!" Notifications It's the website and the standard UI translations.

So I can manage the english quotes and the translators can translate them but do you want it done in the repository and just add through pull requests (I can @ you to clear them and merge you) or should I make a different repository. 

I thought for the guidelines:
1. Reasonably clean because kids can use Manuksript too. (I doubt there will be many quotes with bad words in them but we can always censor)
2. Keep it generally about writing and not limited to only one genre. I was considering adding some about more suspific things like Sanderson's laws of magic because they apply to most bits of world building. But I think you get the point.
3. They must have the source of who there from.

There won't be images because of copyrights but just quotes on some sort of banner (Made by the UI for easy scalability.)

Anyone have anything else they want to add?
